recipes/vega/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+$PYTHON setup.py install
+
+# Add more build steps here, if they are necessary.
+
+# See
+# http://docs.continuum.io/conda/build.html
+# for a list of environment variables that are set during the build process.
```

Comment:
You can remove both `build.sh` and `bld.bat` now. I see you already have the `script` section in the `meta.yaml` file.